src/build/release.rs

```diff
@@ -15,7 +18,12 @@ pub fn release(p: &crate::project::Project, version: &String) -> Result<(), Erro
         fs::create_dir_all(format!("releases/{}/@{}/keys", version, modname))?;
     }
     for file in &p.files {
-        fs::copy(file, format!("releases/{}/@{}/{}", version, modname, file))?;
+        for entry in glob(file).unwrap_or_print() {
+            if let Ok(path) = entry {
+                let file_name = path.file_name().unwrap();
+                fs::copy(&path, format!("releases/{}/@{}/{:?}", version, modname, file_name))?;
```

Comment:
Avoid the use of the Debug formatting. Should be converted like done [here](https://github.com/synixebrett/HEMTT/blob/57cdb4257ce8bd9bcc36cb96a5cfcc7c52b3caec/src/build/script.rs#L141)